skops/utils/fixes.py

```diff
@@ -20,38 +18,3 @@
     # if you're removing this, you should also remove the dependency from
     # _min_dependencies.py
     from typing_extensions import Literal  # noqa
-
```

Comment:
The lines above, importing `metadata` and `Literal`, can also be deleted, no? Hence the whole of `fixes.py`, hence the whole of `skops/utils/`.